tests/StacktraceTest.php

```diff
@@ -96,4 +96,27 @@ public function testWeCanHandleLaravelBasedExceptions()
         $this->assertSame('App\Domain\Service\GitHub->__construct(Object(App\Social))', $firstCodeframe->frame);
         $this->assertCount(0, $firstCodeframe->code);
     }
+
+    public function testWeCanHandlePDOBasedExceptions()
+    {
+        $exceptionString = <<<EOS
```

Comment:
I'm curious, why you're using `<<<EOS` in place of what I have used elsewhere in the tests. 🤔 

Any chance I could bother you to explain the difference in your opinion 😄

Reply:
I would really appreciate it if we could keep the same style 😄

Reply:
Maybe something like 
```
        $exceptionString = "PDOException: SQLSTATE[22008]: Datetime field overflow: 7 ERROR:  date/time field value out of range: \"1967-12-0\"\nHINT:  Perhaps you need a different \"datestyle\" setting. in /home/austinkregel/Sites/lager/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:142\nStack trace:\n#0 /home/austinkregel/Sites/lager/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php(142): PDOStatement->execute(NULL)\n#1 /home/austinkregel/Sites/lager/vendor/laravel/framework/src/Illuminate/Database/Connection.php(330): Doctrine\DBAL\Driver\PDOStatement->execute()\n#2 /home/austinkregel/Sites/lager/vendor/laravel/framework/src/Illuminate/Database/Connection.php(657): Illuminate\Database\Connection->Illuminate\Database\{closure}('insert into \"ta...', Array)";
```

Reply:
Fixed in c92215d